python/py-tensorflow-macos/Portfile

```diff
@@ -39,11 +39,14 @@ platform i386 {
     set minimum_supported_major_version 20
 }
 
-if {${os.platform} eq {darwin} && ${os.major} < ${minimum_supported_major_version}} {
+if {!(${os.arch} in [list arm i386])
```

Comment:
this should be done correctly in the first commit that touches the `py-tensorflow-macos` port; now you change it again in the second commit....

Reply:
Fixed thanks.